src/Shouldly/Configuration/KnownDoNotLaunchStrategies.cs

```diff
@@ -21,8 +21,8 @@ public class KnownDoNotLaunchStrategies
         public readonly IShouldNotLaunchDiffTool MyGet = new DoNotLaunchWhenEnvVariableIsPresent("BuildRunner");
         [UsedImplicitly]
         public readonly IShouldNotLaunchDiffTool TravisCI = new DoNotLaunchWhenEnvVariableIsPresent("TRAVIS");
-        [UsedImplicitly]
-        public readonly IShouldNotLaunchDiffTool NotWindows = new DoNotLaunchWhenPlatformIsNotWindows();
+        //[UsedImplicitly]
+        // public readonly IShouldNotLaunchDiffTool NotWindows = new DoNotLaunchWhenPlatformIsNotWindows();
```

Comment:
I immediately regret not calling this `DoNotLaunchWhenPlatformIsNotSupported`, and then we could have changed this to a no-op without having to change the api 😞